conjure-java-core/src/integrationInput/java/com/palantir/another/ServerConjureErrors.java

```diff
@@ -0,0 +1,28 @@
+package com.palantir.another;
+
+import com.palantir.conjure.java.undertow.lib.CheckedServiceException;
+import javax.annotation.Nullable;
+import javax.annotation.processing.Generated;
+
+@Generated("com.palantir.conjure.java.types.CheckedErrorGenerator")
+public final class ServerConjureErrors {
+    private ServerConjureErrors() {}
+
+    public static DifferentPackage differentPackage() {
+        return new DifferentPackage();
+    }
+
+    public static DifferentPackage differentPackage(@Nullable Throwable cause) {
+        return new DifferentPackage(cause);
+    }
+
+    public static final class DifferentPackage extends CheckedServiceException {
+        private DifferentPackage() {
+            super(com.palantir.another.ConjureErrors.DIFFERENT_PACKAGE);
+        }
+
+        private DifferentPackage(@Nullable Throwable cause) {
+            super(com.palantir.another.ConjureErrors.DIFFERENT_PACKAGE, cause);
+        }
+    }
+}
```

Comment:
I suspect this file can be deleted, it appears to have been generated with an older version of the codegen (no throwIf util functions, and still has two ctors for the exception)

Reply:
Sorry, you're right! Will remove.